doc/source/whatsnew/v3.0.0.rst

```diff
@@ -35,7 +35,7 @@ Other enhancements
 - Support passing a :class:`Series` input to :func:`json_normalize` that retains the :class:`Series` :class:`Index` (:issue:`51452`)
 - Users can globally disable any ``PerformanceWarning`` by setting the option ``mode.performance_warnings`` to ``False`` (:issue:`56920`)
 - :meth:`Styler.format_index_names` can now be used to format the index and column names (:issue:`48936` and :issue:`47489`)
--
+- :meth:`DataFrame.fillna` and :meth:`Series.fillna` can now accept ``value=None``; for non-object dtype the corresponding NA value will be used (:issue:`57723`)
```

Comment:
The other change here is that you must specify a value for `value`, whereas before it defaulted to `None`

Reply:
Previously, the default value of None would also raise. In other words, both before an after this PR you had to specify `value`.

Reply:
> Previously, the default value of None would also raise. In other words, both before an after this PR you have to specify `value`.

That was true if you didn't specify `method`.  But with 2.2:
```python
>>> import pandas as pd
>>> import numpy as np
>>> s=pd.Series([1,np.nan,2])
>>> s.fillna(method="ffill")
<stdin>:1: FutureWarning: Series.fillna with 'method' is deprecated and will raise in a future version. Use obj.ffill() or obj.bfill() instead.
0    1.0
1    1.0
2    2.0
dtype: float64
>>>
```
So I think it is worth saying that `value` is now required.

Reply:
But method is no longer an argument. Any code that does not raise a warning in 2.2 must be specifying an the `value` argument. Any code that raises a warning in 2.2 will raise a TypeError in 3.0.

Reply:
So don't we have to say somewhere that we are enforcing the deprecation that `method` is no longer an argument?

Reply:
https://github.com/pandas-dev/pandas/blob/6e09e97c10ad7591ffcc7d638075bdf20de8294d/doc/source/whatsnew/v3.0.0.rst?plain=1#L283

Reply:
Great.  All good then